<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Karaoke</class>
 <widget class="QMainWindow" name="Karaoke">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Karaoke</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="CDG" name="cdgDisplay" native="true"/>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnFile">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPlay">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&gt;</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CDG</class>
   <extends>QWidget</extends>
   <header>widgets/cdg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnPlay</sender>
   <signal>clicked(bool)</signal>
   <receiver>Karaoke</receiver>
   <slot>playPushed(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnFile</sender>
   <signal>clicked()</signal>
   <receiver>Karaoke</receiver>
   <slot>selectFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>testPushed()</slot>
  <slot>playPushed(bool)</slot>
  <slot>selectFile()</slot>
 </slots>
</ui>
